keras/ops/numpy_test.py

```diff
@@ -3930,7 +3956,7 @@ def test_add(self, dtype1, dtype2):
             standardize_dtype(jnp.add(x1_jax, x2_jax).dtype),
         )
 
-    @parameterized.parameters(INT_DTYPES)
+    @parameterized.named_parameters(named_product(dtype=INT_DTYPES))
```

Comment:
Why would it be necessary to use `named_product` if there is only one iterable?

Reply:
To automatically name the tests. Instead of:
* `test_add1`
* `test_add2`
* ...
You get:
* `test_add_int32`
* `test_add_int64`
* ...

One alternative is
```
@parameterized.named_parameters({"testcase_name": str(t), "dtype":t} for t in INT_DTYPES)
```
which works but is not super readable and doesn't fit in one line.